examples/add_images.py

```diff
@@ -0,0 +1,25 @@
+"""
+This example shows two image layers using the add_image API
+"""
+
+import sys
+from PyQt5.QtWidgets import QApplication
+
+from skimage import data
+from skimage.color import rgb2gray
+from napari_gui import Window, Viewer
+
+
+if __name__ == '__main__':
```

Comment:
My guess is that we don't need the `__main__` guards for these scripts. We don't have any in the scikit-image examples, because they don't define any functions, so no one would ever want to import them. (Famous last words in general, but indeed in this case that's how it's played out.)

Reply:
Sounds good, I have removed them